content/nextflow-on-aws-batch/35_nextflow201/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Local Run w/ AWS Batch"
+chapter: true
+weight: 35
+---
+
+## Nextflow on AWS Batch
+
+Now that we setup AWS Batch we can use Nextflow to submit jobs we are getting closer to our architecture.
+
+The local run will use the job-queue to submit jobs while the nextflow process is running locally.
```

Comment:
No need for dash '-' between job and queue.
And capitalize Nextflow

Reply:
done